docs/howto/how_to_leverage_scenario_editor.md

```diff
@@ -21,11 +16,13 @@ In order to use this feature, please send an email to our BD team at idg-apollo@
 
     ![](images/se_location1.png)
 
-3. Once inside, you should see the prompt as seen in the image below, requesting you to send us an email from your company's email. As this app is in Beta testing, it is not open to all our developers.
+3. Once inside, you will have to comeplete the form on the screen as seen in the image below. As this app is in Beta testing, it is not open to all our developers.
 
-    ![](images/login_se.png)
+    ![](https://github.com/ApolloAuto/apollo/blob/master/docs/specs/images/form.png)
 
-4. In 3 business days, you should receive an email confirming your account's approval and activation, after which, your screen should no longer display the prompt and the buttons would no longer be greyed out.
+4. You should receive the following activation confirmation via email:
```

Comment:
maybe still keep the "within 3 business days"?

Reply:
Done